src/spyglass/lfp/v1/lfp.py

```diff
@@ -58,7 +60,7 @@ class LFPV1(SpyglassMixin, dj.Computed):
     """
 
     def make(self, key):
-        DECIMATION_FACTOR = 1000
+        DECIMATION_FACTOR = key["lfp_sampling_rate"]
```

Comment:
I would delete this line and change line 96 to:
```python
decimation = sampling_rate // key["lfp_sampling_rate"]
```

Also remove the comment that says it is targeted a 1 kHz sampling rate